jekyll/_cci2/webhooks.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: classic-docs
+title: "Webhooks"
+short-title: "Using Webhooks to subscribe to CircleCI events"
+description: "Using Webhooks to subscribe to CircleCI events"
+version:
+- Cloud
+---
+
+## Overview
+{: #overview}
+
+A webhook, sometimes referred to as a _reverse API_, allows you to connect a
+platform you manage (either an API you create yourself, or a third party
+service) to a stream of _future events_ (or, _real-time information_).
+
+Setting up a Webhook on CircleCI enables you to receive information (referred to
+as _events_) from CircleCI, as they happen. This can help you avoid polling the
+API or manually checking the CircleCI web application for desired information.
+
+The rest of this document will detail how to set up a webhook as well as the
+shape of events that will be sent to your webhook destination.
+
+**Note:** The Webhooks feature on CircleCI is currently in preview; documentation and features may change or be added to.
+
+## Setting up a hook
+{: #setting-up-a-hook}
+Webhooks are set up on a per-project basis. To get started:
+
+1. Visit a specific project you have setup on CircleCI.
+1. Click on **Project Settings**.
+1. In the sidebar of your Project Settings, click on **Webhooks**.
+1. Click **Add Webhook**.
+1. Fill out the Webhook form (the table below describes the fields and their intent):
+
+| Field                  | Required? | Intent                                                               |
+|------------------------|-----------|----------------------------------------------------------------------|
+| Webhook name           | Y         | The name of your webhook                                             |
+| URL                    | Y         | The URL the webhook will make POST requests to.                      |
+| Certificate Validation | Y         |                                                                      |
+| Secret token           | Y         | Used by your API/platform to validate incoming data is from CircleCI |
+| Select an event        | Y         | You must select at least one event that will trigger a webhook       |
+{: class="table table-striped"}
+
+## Event Specifications
+{: #event-specifications}
+
+CircleCI currently offers webhooks for the following events:
+
+| Event type         | Description                                              | Potential statuses                                       | Included sub-entities                          |
+|--------------------|----------------------------------------------------------|----------------------------------------------------------|------------------------------------------------|
+| ping               | Test ping event when the webhook is initially configured |                                                          | project, organization                          |
```

Comment:
Ping isn't a webhook event - it's just a test you can do once a specific webhook has been set up. 

What if we made "Test your Webhook" its own section below Event Specifications, and briefly describe how to use ping there?

Reply:
Good point. I added an additional step about dispatching a test ping event under "Setting up a hook"